Frameworks/MQTTnet.NetStandard/Implementations/MqttTcpChannel.cs

```diff
@@ -73,11 +74,12 @@ public Task DisconnectAsync()
         public Task<int> ReadAsync(byte[] buffer, int offset, int count, CancellationToken cancellationToken)
         {
             return _stream.ReadAsync(buffer, offset, count, cancellationToken);
+
```

Comment:
?

Reply:
Trash indeed, guess I was sleepy at that time.